dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
@Ana06 

> in a different test suite called something like database structure?

A new test suite means a new build for travis. Let's not overdo it just for a one-liner...